### ISSUE TITLE
Broken Depth Buffer

### ISSUE THREAD

**Issue by Exeonz** [SaiyansKing/GD3D11#9]
Depth buffer is broken when using Reshade so RTGI is not compatible with this fork. It worked perfectly fine before with dev20.

**Comment by SaiyansKing** [comment]
And what that have to do with GD3D11 modification? You're using 3rd party modification so if you've problem with it ask the author of that modification to fix it, it isn't my responsibility to make it work.
If there is a problem with ReShade detecting depth buffer you should rather contact author of ReShade.
If there is a problem with 3rd party shader ask author of that shader to fix it.

I'll close instantly any issue that isn't related to the GD3D11 project so keep that in mind next time.

**Comment by Exeonz** [comment]
First of all, how rude I was very polite to you and you give me a response like this, ok.
Every game has depth buffer, game wouldn't work without it. For gothic games GD3D11 serves as a wrapper due to hold old the native renderer is. Broken depth buffer has nothing to do with Reshade itself and as I pointed out before previous version of GD3D11 dev20 made by Kirides never has this issue so it's your fork that's breaking it, not Kirides and not Reshade. Again it's not Reshade's fault for not detecting your depth buffer, it is being detected but it isn't working because it's broken and depth buffer isn't a shader and isn't a part of reshade. Reshade and RTGI by Pascal Gilcher has the biggest modding community with thousands of people in discord alone, which I have made multiple guides for modding the game and getting RTGI to work so I thought you should know that depth buffer in your specific fork is not working. If you refuse to fix your own mod and give me a rude response like this I will just simply point all those people to a working version of GD3D11 instead.

**Comment by SaiyansKing** [comment]
1. I'm not rude, only stating that I don't want to hear about issues caused by third-party software.
2. Show me an clear image of broken depth buffer and when it happen(without third party software), e.g. you can see object x through wall.
3. I'll gladly fix anything related to GD3D11(assuming my knowledge will allow me) as long as it doesn't happen by third party software.
4. I don't care about rude people such as yourself using anything written by me, it is you who appear here saying something is wrong without showing any proof and specified what is exactly wrong. If the depth buffer is not working then you would get crappy mess instead of an clear image from game where you can see "depth" and somehow it is possible to play the game without working "depth buffer"(I find it rather interesting).

**Comment by Exeonz** [comment]
I'm not trying to sound rude and if I originally came off that way I apologize. All I want is for your fork to display depth buffer, which is a problem for the modding community if it doesn't due to making allot of things incompatible. I'm not a GD3D11 developer, all I can say is that somewhere along your update it broke depth buffer from displaying/being able to be read using Reshade, Gshade or any other post processing injector. I know that to be the case because dev21 doesn't display it when dev20 is working perfectly fine. I can provide you screenshots or any other info but like I've stated I'm not a GD3D11 developer I don't know what exactly is causing this issue. My point here is that depth buffer is not being read/displayed and you're telling me to show you ingame how it's not working, well that's not the point I'm making. Also the term "Broken Depth Buffer" comes from RTGI community meaning that something within renderer's API is preventing depth buffer from being able to be read/displayed, doesn't mean that the game is missing it in the literal sense
![dev33 no depth buffer](https://user-images.githubusercontent.com/95699217/145131892-f9c2c4bd-24db-4263-9aaf-11709efcf093.png)
![dev20 depth buffer working](https://user-images.githubusercontent.com/95699217/145131914-84714737-b254-459b-9ddd-9eb97322f06f.png)
.

**Comment by Exeonz** [comment]
Great, it's working for you. I don't know which GD3D11 Version you're using or what you did to make it work but it wasn't working for me or anybody else who was trying to make it work today. There's also allot of people asking about it on https://forum.worldofplayers.de/forum/threads/1546222-Yet-Another-D3D11-Renderer/page50

**Comment by Exeonz** [comment]
I found a workaround to get depth buffer to display for RTGI on dev33 (I got normal map to display for DisplayDepth prior). It's a long story so I won't go into detail but tldr after editing allot of values including dx11 and changing game values I had to add new line and multiply depth buffer by 50000 for it to show up for RTGI, maybe something could be done about it, I've never seen any other game do that including prior dev releases. (dev20 was just enabling copy depth buffer and selecting it.) Everything is working now, sorry again if I came off rude or annoying and thank you for your contribution to these games.
